helm/adelphi/templates/cassandra-image-builder.yaml

```diff
@@ -10,20 +10,33 @@ spec:
   - name: cassandra-image-template
     inputs:
       parameters:
-      - name: registry_ip
       - name: git_identifier
     container:
-      image: gcr.io/kaniko-project/executor:v1.0.0
+      image: gcr.io/kaniko-project/executor:v1.6.0-debug
+      command:
+        - sh
+        - -c
       args:
-      - --dockerfile=/dockerfile/Dockerfile
-      - --context=dir:///workspace
-      - {{`--destination={{inputs.parameters.registry_ip}}:30000/adelphi/cassandra`}}
-      - --insecure=true
+        - until `wget --spider http://localhost:30000`; do echo sleep && sleep 1; done;
```

Comment:
The `--spider` flag will perform a HEAD request instead of a GET just to check if the proxy is up and running, it won't care about the respond body.